email_alert_notifications/jenkins.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+export FILE_TO_UPLOAD=rename_email_files_with_request_id.py.zip
+mkdir -p archive/$(date +%s)
+mv $FILE_TO_UPLOAD archive/$(date +$s)/
+/usr/bin/zip $FILE_TO_UPLOAD rename_email_files_with_request_id.py
```

Comment:
Do lines 6 and 7 need to be the other way round? I think it's trying to `mv` a file that doesn't exist at the moment.